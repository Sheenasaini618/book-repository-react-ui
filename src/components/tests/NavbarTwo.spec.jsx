import React from 'react';
import '@testing-library/jest-dom';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import NavbarTwo from '../NavbarTwo';

configure({ adapter: new Adapter() });

global.fetch = jest.fn().mockImplementation(() =>
    Promise.resolve({
        ok: true,
        status: 200,
    }),
);

describe('<NavbarTwo />', () => {

    let props;
    let component;

    beforeEach(() => {

        props = {
            setData : jest.fn(),
            data : ""
        }

        component = shallow(<NavbarTwo {...props} />);

    })

    it('should match snapshot', () => {
        expect(component.debug).toMatchSnapshot();
    });
})