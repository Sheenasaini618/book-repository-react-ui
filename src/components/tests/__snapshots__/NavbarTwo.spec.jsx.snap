// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavbarTwo /> should match snapshot 1`] = `[Function]`;
